pkg/adafruitio/adafruitio.go

```diff
@@ -27,15 +28,15 @@ func New(username, AIOKey string) (*Client, error) {
 	// Check client credentials.
 	req, err := http.NewRequest(http.MethodGet, "https://io.adafruit.com/api/v2/user", nil)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "could not construct API request")
```

Comment:
since we might have additional APIs we're making requests to in the future, maybe we should be more specific and make it "could not construct adafruit API request"? ditto for all other wrapped errors in this package. your call.

Reply:
Let's figure this out when we get there.